### ISSUE TITLE
Plots showing results in Colab notebook display metrics on different scales.

### ISSUE THREAD

**Issue by mc-robinson** [OATML/bdl-benchmarks#1]
I could be wrong, but this plot for example

![image](https://user-images.githubusercontent.com/10458975/60199005-d4f71b00-983a-11e9-94de-d5b4f076f94a.png)

is clearly suspect since the naive method is giving an AUC around 0 and not 50. I believe the issue is that the `results` dictionary gives the metrics on a scale from [0,1], while the baseline models give the metrics on a scale of [0,100]. I can't actually find in the code where these baseline results are scaled by 100, or else I would happily make a pull request. In general, I think it would be easier to keep everything on a scale of [0,1]

**Comment by filangelos** [comment]
Thanks for your issue @mc-robinson! You are right, the [`baselines`](https://github.com/OATML/bdl-benchmarks/tree/alpha/leaderboard/diabetic_retinopathy_diagnosis) results are in the range [0, 100], while the `tf.keras.metrics` for [AUC](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/metrics/AUC) and [Accuracy](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/metrics/Accuracy) in the range [0, 1]. We are fixing it in the next release, by rescaling baselines in [0, 1].
